oneview_redfish_toolkit/blueprints/subscription_collection.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import logging
+
+
+from flask import abort
+from flask import Blueprint
+from flask import Response
+from flask_api import status
+
+from oneview_redfish_toolkit.api.subscription_collection \
+    import SubscriptionCollection
+
+subscription_collection = Blueprint("subscription_collection", __name__)
+
+
+@subscription_collection.route(
```

Comment:
I haven't found any info about this but what is supposed to happen when we try to access/modify this collection in a scenario that `EventService['ServiceEnabled'] == False`????? Should we fail all requests?

Reply:
I didn't find any info too, but fail all requests when `ServiceEnabled == False` looks good for me.

Reply:
@ricardoas In that case of fail the requests, should I do a GET operation by the `EventService` into `SubscriptionCollection` to check if `ServiceEnabled == False`???

Reply:
I guess that if we have this logic isolated in a function in util we can bypass the GET operation and just call the method directly... wdyt?

Reply:
Sounds good!